samples/n5010-test/n5010-test.c

```diff
@@ -559,42 +569,50 @@ static fpga_result fpga_test_qdr(struct n5010 *n5010)
 	res = fpgaWriteMMIO64(n5010->handle, 0, n5010->base + QDR_TEST_CTRL, 1);
 	if (res != FPGA_OK) {
 		fprintf(stderr, "failed to start test: %s\n", fpgaErrStr(res));
+		if (res == FPGA_OK)
```

Comment:
Could we fix this one?

Reply:
Certainly. Good catch. I had put it in the wrong if that did not have the "|| stat != 0"